SofaKernel/modules/Sofa.Geometry/src/sofa/geometry/Tetrahedron.h

```diff
@@ -54,23 +53,14 @@ struct Tetrahedron
     static constexpr auto volume(const Node& n0, const Node& n1, const Node& n2, const Node& n3)
     {
         constexpr Node n{};
-        //static_assert(std::distance(std::begin(n), std::end(n)) == 3, "volume can only be computed in 3 dimensions.");
+        static_assert(std::distance(std::begin(n), std::end(n)) == 3, "volume can only be computed in 3 dimensions.");
```

Comment:
should'nt it be a enable_if here ?

Reply:
IMO the static_assert version will give a more meaningful message when throwing a compilation error (in my experience, std::enable_if is a bit... confusing when the compilation fails)